pom.xml

```diff
@@ -137,6 +137,7 @@
     <properties>
         <jdk.version>11</jdk.version>
         <mvn.version>3.2.5</mvn.version>
+        <project.build.outputTimestamp>2023-04-09T08:38:05Z</project.build.outputTimestamp>
```

Comment:
Why do we set this in projects? Since EE4J.project 1.0.8 it is inherited: https://repo1.maven.org/maven2/org/eclipse/ee4j/project/1.0.8/project-1.0.8.pom
I think we just should not use the value, so the includeDate set to false should be enough. Or am I wrong?

Reply:
each build should define its own property to have its time adapted to its own release cycle, instead of having a value inherited from parent

Reply:
Aha, it is used by plugins. https://maven.apache.org/guides/mini/guide-reproducible-builds.html